index.js

```diff
@@ -42,6 +43,9 @@ class ClinicDoctor extends events.EventEmitter {
     this.detectPort = detectPort
     this.debug = debug
     this.path = dest
+
+    // cannot calculate ELU on these node versions
+    this.collectLoopUtilization = !isOneOfNodeVersions(['12', '14'])
```

Comment:
```suggestion
    this.collectLoopUtilization = !isOneOfNodeVersions(['12', '14'])
```
I would change it to `semver.gt('16.0.0', process.version)` to avoid complications using non LTS versions like v13, v15